lib/github-auth.js

```diff
@@ -2,14 +2,15 @@ var querystring = require('querystring');
 var request = require('request');
 var autosave = require('json-autosave');
 var serverSecrets;
+var baseUrl = process.env.BASE_URL || "https://shields.io";
```

Comment:
Or we go pragmatic and use `BASE_URL` for both. Just arguing with myself here...